src/main/java/uk/gov/hmcts/reform/professionalapi/controller/advice/LogAndSuppressRequestRejectedExceptionFilter.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.professionalapi.controller.advice;
+
+import com.microsoft.applicationinsights.core.dependencies.google.gson.Gson;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpHeaders;
+import org.springframework.security.web.firewall.RequestRejectedException;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.GenericFilterBean;
+
+@Component
+@Slf4j
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class LogAndSuppressRequestRejectedExceptionFilter extends GenericFilterBean {
```

Comment:
does this need to be of highest precedence? can you try with a lower or default precedence?